### ISSUE TITLE
Consider pulling Spandx into the SPAship family

### ISSUE THREAD

**Issue by Jared-Sprague** [spaship/spaship#78]
@mwcz Was talking to Chris and others during the trip and there is some desire to make Spandx part of the SPAship suite of tools for frontend devs.  This issue is to discuss the pros and cons of pulling in Spandx under the SPAship umbrella.

**Comment by mwcz** [comment]
I think that's a good idea.  

1. We can move spandx into the spaship monorepo
1. We can keep publishing it with the package name `spandx` to maintain consistency (ie, we should not publish it under `@spaship/spandx`)
1. Having spandx in the monorepo will make it easier to test the spandx+spaship integration, such as teaching spandx to understand spaship.yaml files
1. Should we consider merging spandx into the spaship cli?

The only downside I can think of is that we already have a bunch to do, and moving spandx into the monorepo will require a modest amount of work.  I think this should be post-MVP.

**Comment by sayak-sarkar** [comment]
I like the idea of eventually merging spandx into the spaship cli, though considering the amount of time that it might require, maybe looking into it post-MVP might make the most sense. I think whether we should move spandx into the spaship monorepo for now or not should ultimately be @mwcz's decision.

**Comment by mwcz** [comment]
I agree, it would require way too much time.  It would be nice, but it's a very low priority.
